src/scheduler/constant.h

```diff
@@ -314,6 +314,8 @@ enum fairshare_flags
 	FS_TRIM = 1
 };
 
+#define FAIRSHARE_MIN_USAGE 1
```

Comment:
Modify new_group_info to use this macro instead of initializing usage and temp_usage to 1.

Reply:
I didn't realize Bhroam had given the same comment :)